README.md

```diff
@@ -294,6 +299,27 @@ delta a.txt b.txt
 diff -u a.txt b.txt | delta
 ```
 
+#### Environment
+
+##### a) Delta
+Delta respects the setting of some environment variables to derive the default pager to use. These are in descending order of priority :
+- `DELTA_PAGER`,
+- `BAT_PAGER` and
+- `PAGER`
+
+Hence if e.g. `$DELTA_PAGER` is unset, delta will use `$BAT_PAGER`, or `$PAGER` if that is unset, too. In environments wthout any of the three variable set, delta's fallback is `less`.
+
+_**NOTE:** Be aware that setting one of those environment variables to `delta` while leaving the ones with higher priority unset (or also pointing at `delta`) can lead to an infinite cycle of forked processes by delta recursively calling itself._
+
+##### b) Third party apps
+Note that `$BAT_PAGER` (as well as `$PAGER`) is _also_ used by the standalone `bat` app. Please see `bat(1)` for a description. However, `$BAT_THEME`, `$BAT_STYLE` and `$BAT_CONFIG_PATH` are **not** regarded by delta.
+
+The behavior of delta's default pager, less, can be controlled using the `LESS` environment variable. It may contain any of the `less` command line options and/or interactive less-commands (prefixed by a leading `+` sign) which will be executed every time right after less is launched. A real-life example for the latter would be `LESS='+Gg'`, causing the pager to briefly jump to the bottom of the buffer and back to the top right afterwards. The purpose of this example is to enable the full metadata display in less' status line, including the buffer contents length and relative position of the current view port, in percent and lines/characters.
+
+Less also uses a bunch of other environment variables as well as config files, although most of them are not relevant for use in combination with delta. For an in-depth explanation of these configuration options please see the `less(1)` manual.
```

Comment:
It would be really nice to link to an online less manual here. Honestly, I consider myself decent with Unix command line stuff and I've never learned how to use `man` or what the numbers really mean. And we do have Windows users of course.

Reply:
I couldn't find one at the upstream site, but I guess we can link to linux.die.net ?

Reply:
>  I guess we can link to linux.die.net 

Yes, I agree that's a well-known resource. Maybe we don't need detail here on `LESS` if we can link to good external docs (I'd even link to a random article / blog post on configuring less if there were a good one, seeing as there are so many possibilities).